### PR TITLE
add account token (for stripe connect)

### PR DESCRIPTION
**Pull request by romainhill**
https://stripe.com/docs/api#create_account_token

### CODE REVIEW

**Comment by jamonholmgren** [comment]
@brentvatne Is this something you'd consider merging?

File: __tests__/Stripe-test.js [hunk 1]
```diff
@@ -53,6 +53,25 @@ describe("PII token creation with valid PII details", function () {
   });
 });
 
+describe("account token creation with valid account details", function () {
+  var accountDetails = {
```
Comment (brentvatne) [inline __tests__/Stripe-test.js:57]:
```suggestion
  let accountDetails = {
```

File: __tests__/Stripe-test.js [hunk 2]
```diff
@@ -53,6 +53,25 @@ describe("PII token creation with valid PII details", function () {
...
+          first_name: 'Jane',
+          last_name: 'Doe',
+        },
+        tos_shown_and_accepted: true,
+    }
+  };
+
+  fetchMock.mock('*', { });
+  wrap.it ('should receive a valid response', async () => {
+    var token = await stripe.createToken(accountDetails);
```
Comment (brentvatne) [inline __tests__/Stripe-test.js:69]:
```suggestion
    let token = await stripe.createToken(accountDetails);
```

File: __tests__/Stripe-test.js [hunk 3]
```diff
@@ -53,6 +53,25 @@ describe("PII token creation with valid PII details", function () {
   });
 });
 
+describe("account token creation with valid account details", function () {
```
Comment (brentvatne) [inline __tests__/Stripe-test.js:56]:
```suggestion
describe("account token creation with valid account details", () => {
```

File: __tests__/Stripe-test.js [hunk 4]
```diff
@@ -53,6 +53,25 @@ describe("PII token creation with valid PII details", function () {
...
+        legal_entity: {
+          first_name: 'Jane',
+          last_name: 'Doe',
+        },
+        tos_shown_and_accepted: true,
+    }
+  };
+
+  fetchMock.mock('*', { });
+  wrap.it ('should receive a valid response', async () => {
```
Comment (brentvatne) [inline __tests__/Stripe-test.js:68]:
```suggestion
  wrap.it('should receive a valid response', async () => {
```

File: __tests__/Stripe-test.js [hunk 5]
```diff
@@ -53,6 +53,25 @@ describe("PII token creation with valid PII details", function () {
   });
 });
 
+describe("account token creation with valid account details", () => {
```
Comment (brentvatne) [inline __tests__/Stripe-test.js:56]:
```suggestion
describe('account token creation with valid account details', () => {
```

**Comment by jeff-da** [comment]
Thanks @brentvatne @romainhill !

**Comment by jamonholmgren** [comment]
I don't think we need this anymore (cc @carlinisaacson) but it would probably still be a good thing to merge.

**Pull request merged**